source/game/Game.gd

```diff
@@ -32,12 +32,19 @@ func _process_factory_loop_volume() -> void:
 	var volume := 0.0
 
 	var miners := get_tree().get_nodes_in_group("Miner")
+	var refineries := get_tree().get_nodes_in_group("Refinery")
+
 	for miner in miners:
 		var distance = camera_position.distance_to(miner.global_position)
-		var temp = clamp(miner_sfx_max_volume - distance / miner_sfx_radius, 0, 1)
+		var temp = clamp(1 - distance / miner_sfx_radius, 0, 1)
+		volume = max(volume, temp)
+
+	for refinery in refineries:
```

Comment:
wouldnt `for building in miners + refineries` do the trick?

Reply:
I gotta check. I'm not aware that you can add arrays like that, but if that works that'd be great!